### DISCUSSION TITLE
Bot not working for some users

### DISCUSSION THREAD

**Discussion by JekaSylar** [westacks/telebot#80] [Q&A]
I can’t understand what could be the problem for some users the bot doesn’t work, that is, they press / start and the reaction is zero, I tested everything works for me.
Can you tell me where to dig
php artisan telebot:webhook --setup
Success! Webhook has been set for 'admar' bot!

**Comment by JekaSylar** [comment]
php artisan telebot:webhook --info                                
************************
*     Webhook Info     *
************************
                                                                                                  
+------------------------+------------------------------------------------------------------------
-------------------+                                                                              
| Bot: admar                                                                                      
                   |
+------------------------+------------------------------------------------------------------------
-------------------+                                                                              
| Key                    | Info                                                                   
                   |
+------------------------+------------------------------------------------------------------------
-------------------+                                                                              
| Url                    | https://admar.com.ua/telebot/webhook/admar/token |                                                                              
| Has Custom Certificate | No                                                                     
                   |                                                                              
| Pending Update Count   | 54                                                                     
                   |                                                                              
| Last Error Date        | 1687285346                                                             
                   |                                                                              
| Last Error Message     | Wrong response from the webhook: 500 Internal Server Error             
                   |                                                                              
| Max Connections        | 40                                                                     
                   |                                                                              
| Ip Address             | 185.166.217.119

**Reply by punyflash** [reply]
You have server error in your code. Check logs
